internal/pkg/source/testdata/src/analyzertest/sourcetest/identification.go

```diff
@@ -14,12 +14,6 @@
 
 package sourcetest
 
-// source container
-type Source struct {
-	Data string // source field
-	ID   int    // public
-}
-
```

Comment:
This file is not particularly large.  Is there a testing reason why you've moved this to a separate test file?

Reply:
I needed the `NewSource` function in two different test files, so I figured it would make more sense to have the `Source` + associated functions in its own file. WDYT?

Reply:
Oh duh.  Works for me.